Makefile

```diff
@@ -76,8 +76,6 @@ LINKER_DIR      = $(ROOT)/src/main/target/link
 # Build tools, so we all share the same versions
 # import macros common to all supported build systems
 include $(ROOT)/make/system-id.mk
-# developer preferences, edit these at will, they'll be gitignored
-include $(ROOT)/make/local.mk
```

Comment:
Why are you removing this line? As the comment states, `make/local.mk` is the designated facility for developers to add their own local settings, and it should be included (if it exists).

Deleting the file itself is the right thing to do, as local settings should not be checked in. (That's why the file is listed in `.gitignore`.)